TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -176,7 +176,7 @@ public static OptionSet GetCommonTypeCobolOptions(TypeCobolConfiguration typeCob
                 { "zcr|zcallreport=", "{PATH} to report of all program called by zcallpgm.", v => typeCobolConfig.ReportZCallFilePath = v },
                 { "dcs|disablecopysuffixing", "Deactivate Euro-Information suffixing.", v => typeCobolConfig.UseEuroInformationLegacyReplacingSyntax = false },
                 { "glm|genlinemap=", "{PATH} to an output file where line mapping will be generated.", v => typeCobolConfig.LineMapFiles.Add(v) },
-                { "cea|checkendalignment=", "Check if a End statement is aligned with the matching opening statement.", v => typeCobolConfig.CheckEndAlignment = !(string.IsNullOrEmpty(v) || v.ToLower() == "false") },
+                { "cea|checkendalignment=", "Check if a End statement is aligned with the matching opening statement.", v => typeCobolConfig.CheckEndAlignment = !(string.IsNullOrEmpty(v) == false && v.ToLower() == "false") },
```

Comment:
Can be simplified as `v?.ToLower() != "false"`.

Reply:
fixed
https://github.com/TypeCobolTeam/TypeCobol/tree/828d955ed380f37e3a26def80e2d534f11fb26d7